docker/README.md

```diff
@@ -53,7 +53,7 @@ It could be necessary to either run all containers in privileged mode, or set se
 
     chcon -Rt svirt_sandbox_file_t data
 
-## Run the Data & WebUI containers
+## Run the data & web UI containers
```

Comment:
Have you tested whether the instructions within that section are still valid?

Reply:
Not done in this commit

Reply:
Discussed with @ilausuch we should just remove the instructions using the outdated image